integration_test/arcilator/JIT/reg.mlir

```diff
@@ -0,0 +1,66 @@
+// RUN: arcilator %s --run --jit-entry=main | FileCheck %s
+// REQUIRES: arcilator-jit
+
+// CHECK: o1 = 10
```

Comment:
Shouldn't this be 16, or am I missing something?

Reply:
https://github.com/llvm/circt/blob/bba8cfb872ae7e1e3263ce77e87ec65305b49b7c/lib/Conversion/ArcToLLVM/LowerArcToLLVM.cpp#L539-L540

It is printed in hex but yeah, it's a bit confusing so I'll change to use a smaller number.

Reply:
Oh, totally forgot about that. Maybe we should prepend a `0x` to this format string. Or a smaller number, a comment, or using `0x10` at the constant op would be nice 🙂